src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -2,0 +1,64 @@
+import classNames from 'classnames';
+import { Todo } from '../../types/todo';
+import usersFromServer from '../../api/users';
+import { UserInfo } from '../UserInfo';
+
+interface TodoInfoProps {
+  todo: Todo;
+}
+
+export const TodoInfo: React.FC<TodoInfoProps> = ({ todo }) => {
+  const { id, title, completed, userId } = todo;
+  const currentUser = usersFromServer.find(user => user.id === userId);
```

Comment:
The logic to find the current user should be moved to the parent component, and the `currentUser` should be passed as a prop to `TodoInfo`. This will decouple the component from direct data fetching and align with best practices.